urgent-care/src/main/java/gov/va/api/health/urgentcare/service/healthcheck/SteelThreadSystemCheck.java

```diff
@@ -4,51 +4,98 @@
 import gov.va.api.health.urgentcare.service.queenelizabeth.client.QueenElizabethClient.QueenElizabethServiceException;
 import gov.va.api.health.urgentcare.service.queenelizabeth.client.Query;
 import gov.va.med.esr.webservices.jaxws.schemas.GetEESummaryResponse;
-import lombok.AllArgsConstructor;
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.actuate.health.Health;
 import org.springframework.boot.actuate.health.HealthIndicator;
-import org.springframework.stereotype.Component;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
 import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.HttpServerErrorException;
 import org.springframework.web.client.ResourceAccessException;
 
-@AllArgsConstructor(onConstructor = @__({@Autowired}))
-@Component
+@Service
 @Slf4j
 public class SteelThreadSystemCheck implements HealthIndicator {
+
   private final QueenElizabethClient client;
 
+  private final SteelThreadSystemCheckLedger ledger;
+
   @Value("${health-check.id}")
   private final String id;
 
+  private final int consecutiveFailureThreshold;
+
+  /**
+   * 'By hand' all args constructor is required to inject non-string values from our properties.
+   *
+   * @param consecutiveFailureThreshold - once consecutive failures meets or exceeds, report
+   *     failures.
+   */
+  public SteelThreadSystemCheck(
+      @Autowired QueenElizabethClient client,
+      @Autowired SteelThreadSystemCheckLedger ledger,
+      @Value("${health-check.medication-id") String id,
```

Comment:
this should be a patient ICN.  Can we change what the variable is in the properties to reflect this?
health-check.patient-icn or something like that